policy-controller/k8s/status/src/index.rs

```diff
@@ -337,3 +365,33 @@ pub(crate) fn make_patch(
     });
     k8s::Patch::Merge(value)
 }
+
+fn now() -> DateTime<Utc> {
+    #[cfg(not(test))]
+    let now = Utc::now();
+    #[cfg(test)]
+    let now = DateTime::<Utc>::MIN_UTC;
+    now
+}
+
+fn no_matching_parent() -> k8s::Condition {
+    k8s::Condition {
+        last_transition_time: k8s::Time(now()),
+        message: "".to_string(),
+        observed_generation: None,
+        reason: "NoMatchingParent".to_string(),
+        status: "False".to_string(),
+        type_: "Accepted".to_string(),
+    }
+}
+
+fn accepted() -> k8s::Condition {
+    k8s::Condition {
+        last_transition_time: k8s::Time(now()),
+        message: "".to_string(),
+        observed_generation: None,
+        reason: "Accepted".to_string(),
```

Comment:
nit, take it or leave it: i would probably want to use a string constant for anything that shows up multiple times, to guard against future typos. not a big deal though.